Issue_Template.md

```diff
@@ -9,7 +9,7 @@ If your issue is not related to the Office Add-ins documentation, please post it
 
 - To report an issue with the Office.js API or platform, create the issue in the OfficeDev/office-js repository (https://github.com/OfficeDev/office-js), which members of the product team monitor for customer-reported issues.
 
-- To submit a feature request for the Office.js API or platform, post your idea to our User Voice page (https://officespdev.uservoice.com/), or if the feature request already exists there, add your vote for it.
+- To submit a feature request for the Office.js API or platform, post your idea to [Microsoft 365 on Q&A](https://docs.microsoft.com/answers/products/m365), or if the feature request already exists there, add your vote for it.
```

Comment:
Should this link follow the same format as the preceding two links, for consistency?